src/meshes/t8code_mesh.jl

```diff
@@ -1096,15 +1106,6 @@ function count_interfaces(forest, ndims)
                                 t8_element_get_linear_id(eclass_scheme, element, max_level)
 
             for iface in 0:(num_faces - 1)
-                pelement_indices_ref = Ref{Ptr{t8_locidx_t}}()
```

Comment:
You can move these variable declarations/initializations in front of the `for` loop. But I expect that Julia optimizes this away. I agree with @ranocha that variables should only be declared inside there scope of usage.